src/DataTable/Actions.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { MoreVert } from '../../icons';
+import {
+  Button, Dropdown, useWindowSize, Icon, breakpoints,
+} from '..';
+
+export const DROPDOWN_BUTTON_TEXT = 'More actions';
+export const SMALL_SCREEN_DROPDOWN_BUTTON_TEXT = 'Actions';
+
+// eslint-disable-next-line react/prop-types
+const CustomToggle = React.forwardRef(({ children, onClick }, ref) => (
```

Comment:
Do we expect `CustomToggle` to always receive an `onClick` prop? If it's not provided (i.e., `'undefined`) without a default prop, calling the function directly within the `onClick` function below will throw an error.

Reply:
Yeah, it's always gonna get an onClick prop, it opens the dropdown.